examples/solidity-contracts/contracts/token/ERC20/IERC20.sol

```diff
@@ -0,0 +1,79 @@
+// SPDX-License-Identifier: MIT
+// OpenZeppelin Contracts (last updated v5.1.0) (token/ERC20/IERC20.sol)
+
+pragma solidity ^0.8.20;
+
+/**
+ * @dev Interface of the ERC-20 standard as defined in the ERC.
+ */
+interface IERC20 {
```

Comment:
I hope we should avoid to copy sol files from solidity library and just use url references

Reply:
The problem is having to deploy `ERC3156FlashBorrowerMock` contract to be able to run in bench for `flash_loan` (see [bench](https://github.com/OpenZeppelin/rust-contracts-stylus/pull/407/files#diff-de72526f0bcd113ba994e7528f578c444fb3278fe03cb9b64f8bb346621528e1R61)); and to be able to implement this contract locally, it seems necessary to have all the dependencies locally as well...

Do you if there's a way to import existing [examples/erc20-flash-mint/tests/mock/borrower.rs](https://github.com/OpenZeppelin/rust-contracts-stylus/pull/407/files#diff-98000cc032e34748af4c97e4dde9fa35b54c7e9f22e1d74950c4ca26c7419247) implementation into benches and reuse it? :thinking: This would make the duplicate here redundant.